ngraph_bridge/ngraph_encapsulate_op.cc

```diff
@@ -405,31 +405,29 @@ void NGraphEncapsulateOp::ComputeUsingParallelExecutor(OpKernelContext* ctx) {
   }
 
   int step_id = ctx->step_id();
-  ngraph::Event event_compile("Compile", "", "");
+  ngraph::Event event_get_ng_item("GetExecutableAndTensors", "", "");
+  shared_ptr<PipelinedTensorsStore> pipelined_tensor_store;
+  std::tuple<int, PipelinedTensorVector, PipelinedTensorVector> io_tensors;
 
-  // Get ngraph executable and inputs information
+  // Get ngraph executable and inputs information and Pipelined tensors
   bool cache_hit;
-  OP_REQUIRES_OK(ctx, m_parallel_executor->GetNgExecutable(tf_input_tensors,
-                                                           ng_exec, cache_hit));
-
+  OP_REQUIRES_OK(
+      ctx, m_parallel_executor->GetExecutableAndTensors(
+               tf_input_tensors, ng_exec, pipelined_tensor_store, cache_hit));
+  io_tensors = pipelined_tensor_store.get()->get_tensors();
+  auto status = Status::OK();
+  if (std::get<0>(io_tensors) < 0) {
+    status = errors::Internal("No free tensor available");
+  }
+  OP_REQUIRES_OK(ctx, status);
```

Comment:
There is another utility OP_REQUIRES use that to throw an error

Reply:
@shresthamalik I used OP_REQUIRES_OK, becuase it is I am getting status, can you please provide 
an example for this particular case on how to use OP_REQUIRES here?